### ISSUE TITLE
Cerebro behind reverse proxy

### ISSUE THREAD

**Issue by TalZiv** [lmenezes/cerebro#98]
Hi,

When I access cerebro through nginx reverse proxy and cerebro is set to / it is working:
`location / {proxy_pass http://localhost:9000;}`

When setting a location like:
`location /cerebro {proxy_pass http://localhost:9000;}`

I get this:
![cerebro](https://cloud.githubusercontent.com/assets/22313897/23856714/44385472-0803-11e7-88aa-d2a2c2a998f6.png)

How can I fix this?

**Comment by triforce** [comment]
This was fixed as part of #48, however there is no current release containing it yet.

**Comment by TalZiv** [comment]
Thanks for the update.

**Comment by lmenezes** [comment]
@TalZiv I have just released a new version that contains this fix, let me know if it works for you :)

**Comment by TalZiv** [comment]
Works great! 
Thank you.

**Comment by iamlennym** [comment]
Hi guys,

My Config:

_# NGINX Config - Upstream_
```
upstream Cluster_cerebro {
        server 10.0.0.12:19000 fail_timeout=0;
}
```

_# NGINX Config - Location_
```
location ~ (^/Consumer_cerebro/) {
        proxy_set_header    Host $host;
        proxy_set_header    X-Real-IP $remote_addr;
        proxy_set_header    X-Forwarded-Proto https;
        proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
        # The rewrite is disabled, to actually have the login screen displayed.
        #rewrite  ^/Consumer_cerebro/(.*)  /$1 break;
        proxy_pass          http://Cluster_cerebro;
}

```

_Cerebro application.conf file_
```
# Application base path
basePath = "/Consumer_cerebro/"
```


I am configuring an instance of cerebro behind NGINX, but I am struggling to get the basePath configured correctly. 

With the above configuration, the login page is displayed correctly, but as soon as I type in my credentials and hit the login button, I get a NGINX 404 Not found with the URL path = **/auth/login**
(without the /Consumer_cerebro/)

If however, I enable the rewrite, it fails without displaying the login screen.

I have already succesfully configured **_kibana_** behind NGINX using its basePath feature, and would have imagined that Cerebro would work the same way. 

**Version of cerebro used: 0.6.3**

Can someone please indicate what I am missing here?   Failing that, I would also settle for a working NGINX configuration that I can _dissect_ myself.

thanks,
Leendert.

**Comment by ezeteze** [comment]
@lmenezes I have the same issue with @iamlennym .
If i enable in application.conf the auth like the following
```
basePath = "/cerebro/"
auth = {
 type: basic
   settings: {
      username = "elastic"
      password = "elastic"
    }
}
```
Then when trying to login it redirects to /auth instead of /cerebro/auth .
nginx configuration like the follwoing
```
  location /cerebro {
    proxy_pass http://127.0.0.1:9000;
  }

```
Only workaround currently is to disable auth.

**Comment by peregrinator** [comment]
Also experiencing the same problem with the 0.6.3 release. It appears as if the action in the login form (app/views/auth/login.scala.html) should not start with a `/`. If you manually edit the form in your browser you can verify this.

@lmenezes any chance of a quick fix/release on that? I'd make a pull request but I'm not completely clear on how to build the project to verify it doesn't break anything else. Thanks!

**Comment by lmenezes** [comment]
@peregrinator this should work now :)

**Comment by peregrinator** [comment]
Thanks for the fast turnaround!

**Comment by sonfrau** [comment]
Good morning,

I'm using last available release: cerebro 0.6.5. I don't have any problems to get cerebro If I visit http://localhost:9000.

I'm trying to use your app behind a NGINX but there's no way.

This is the applied configuration on NGINX (/etc/nginx/sites-available/default):

```
upstream cerebro {
    server localhost:9000 fail_timeout=0;
}

server {
        listen 80;

        location /cerebro {
                proxy_pass http://cerebro/;
        }
}
```
I get this:
![image](https://cloud.githubusercontent.com/assets/22051155/25431138/9fea6726-2a7f-11e7-8571-13249e89209f.png)

It seems like I had problems to get *.css and *.js files.

What could I do to fix this?

Let me know if you need further information.

Thanks and kind regards.

**Comment by sonfrau** [comment]
Hi again,

I got it!

I did this change on :
```
xxx@xxx:~$ cat cerebro-0.6.5/conf/application.conf           
# Secret will be used to sign session cookies, CSRF tokens and for other encryption utilities.
# It is highly recommended to change this value before running cerebro in production.
secret = ""

# Application base path
basePath = "/cerebro/"

```
and this one on /etc/nginx/sites-available/default:
```
upstream cerebro {
    server localhost:9000 fail_timeout=0;
}

server {
        listen 80;

        location /cerebro {
                proxy_pass http://cerebro;
        }
}
```
and I got it to work
